slic-tools/get.js

```diff
@@ -0,0 +1,40 @@
+const { getUserServiceUrl } = require('./url-retriever')
```

Comment:
Think about the module name and what its purpose is. 'get' doesn't say what the module is for.